src/content/docs/auto-telemetry-pixie/manage-pixie-data.mdx

```diff
@@ -0,0 +1,98 @@
+
+```
+  --set newrelic-pixie.excludeNamespacesRegex="default|kube-node-lease|kube-public|kube-system|newrelic|newrelic-custom-metrics|olm|px-operator"
+```
+
+Or, if you wanted to exclude pods running in a non-excluded namespace, you could add another configuration parameter to your Helm install or Helm upgrade.  Instead of matching exact names, you can use a simple regex to match Pod names related to load testing, just as an example. 
+
+```
+
+--set newrelic-pixie.excludePodsRegex="load-test.*|loadgen.*"
+
+```
+  
+If you're performing a brand new install, your Helm command should look something like this:
+
+```
+kubectl apply -f https://raw.githubusercontent.com/pixie-labs/pixie/main/k8s/operator/crd/base/px.dev_viziers.yaml && \
+kubectl apply -f https://raw.githubusercontent.com/pixie-labs/pixie/main/k8s/operator/helm/crds/olm_crd.yaml && \
+helm repo add newrelic https://helm-charts.newrelic.com && helm repo update && \
+kubectl create namespace newrelic ; helm upgrade --install newrelic-bundle newrelic/nri-bundle \
+ --set global.licenseKey=<NR LICENSE KEY> \
+ --set global.cluster=pixie-auto-telemetry \
+ --namespace=newrelic \
+ --set newrelic-infrastructure.privileged=true \
+ --set ksm.enabled=true \
+ --set prometheus.enabled=true \
+ --set kubeEvents.enabled=true \
+ --set logging.enabled=true \
+ --set newrelic-pixie.enabled=true \
+ --set newrelic-pixie.apiKey=<PIXIE API KEY> \
+ --set pixie-chart.enabled=true \
+ --set pixie-chart.deployKey=<PIXIE DEPLOY KEY> \
+ --set pixie-chart.clusterName=pixie-auto-telemetry \
+ --set newrelic-pixie.excludeNamespacesRegex="default|kube-node-lease|kube-public|kube-system|newrelic|newrelic-custom-metrics|olm|px-operator" \ 
+ --set newrelic-pixie.excludePodsRegex="load-test.*|loadgen.*"
+```
+
+If you're just upgrading an existing install, then this is a much simpler approach:
+
+```
+helm upgrade newrelic-bundle newrelic/nri-bundle --reuse-values -n newrelic --set newrelic-pixie.excludeNamespacesRegex="default|k
+```
+
+Learn more about the available parameters for the `newrelic-pixie` Helm chart [here](https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-pixie#chart-details).
+
+
+
+## Use Kubernetes features to collect selected data [#selected-data]
+
+When you deploy Auto-telemetry with Pixie, you're actually enabling the Auto-telemetry with Pixie helm chart, as well as other New Relic components included in the New Relic infrastructure bundle. The [Pixie Edge Module (PEM)](https://docs.pixielabs.ai/tutorials/custom-data/distributed-bpftrace-deployment#background) pods are deployed to the cluster as a Kubernetes DaemonSet, which means that by default, a pod is scheduled to every cluster node and is responsible for collecting all of the observability metrics for that node.
```

Comment:
same comment re: Auto

Reply:
But should helm be Helm?

Reply:
Second sentence in this paragraph is something like 40 words long. Maybe consider breaking into two sentences by changing ", which" to "This means"?